src/applications/disability-benefits/2346/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "appName": "Form-2346",
```

Comment:
This should be something like "Request for batteries and accessories".

Reply:
Done